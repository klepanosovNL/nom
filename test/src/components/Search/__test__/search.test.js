import React from 'react';
import renderer from 'react-test-renderer';
import * as reactRedux from 'react-redux';
import { Search } from '../Search';

describe('Search"s tests', () => {
	beforeEach(() => {
		useSelectorMock.mockClear();
		useDispatchMock.mockClear();
	});

	const useSelectorMock = jest.spyOn(reactRedux, 'useSelector');
	const useDispatchMock = jest.spyOn(reactRedux, 'useDispatch');

	it('Search"s snapshot', () => {
		const tree = renderer.create(<Search></Search>).toJSON();
		expect(tree).toMatchSnapshot();
	});
});
