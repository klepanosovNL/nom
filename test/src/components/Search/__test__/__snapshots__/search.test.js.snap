// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search"s tests Search"s snapshot 1`] = `
<div
  className="search"
>
  <i
    className="fa fa-search"
  />
  <input
    onChange={[Function]}
    onFocus={[Function]}
    placeholder="Filter by name"
  />
</div>
`;
